docs/tutorials.rst

```diff
@@ -22,8 +22,23 @@ start.
 
         pip install -r binder/requirements.txt
 
+    One of the dependencies is `Fiona`_, it is described as "`GDAL`_’s neat and nimble vector API for Python programmers". `GDAL`_ is a 
+    translator library for raster and vector geospatial data formats. Fiona makes it easy to work with. It's required by GeoPandas,
+    which extends the Pandas DataFrames we all love. Combining these three packages, it makes us easy to convert ESRI Shape files to GML files.
+    However, it can be cumbersome to get it installed on a Windows. Please consult the installation guides of the different packages or give
+    the following a try:
+
+    #. Download the latest wheel (choose the approptiate version, based on your python distribution and operating system) 
```

Comment:
I think we discussed the usage of GDAL and geopandas previously. It used to be problematic, see [this link](https://github.com/DOV-Vlaanderen/pydov/issues/132#issuecomment-491781921). Apparently, it works out now. Could it be better to suggest installation using conda? That way you are sure all binaries should match just fine. @jorisvandenbossche, what do you think?

Reply:
sure, but my experience with conda is also limited to simple usage.  
I read that things can be worked out fine with an environment.yml (see [this link](https://github.com/binder-examples/python-conda_pip)).  
Or it can be elaborated with a requirements.txt as discussed [here](https://gist.github.com/luiscape/19d2d73a8c7b59411a2fb73a697f5ed4).  
Depending on the approach, we should also alter the suggestion on our readme with `pip install` to `conda install`.
Of course, all of the above is not required if the issues between GDAL and geopandas are not present anymore.

Reply:
You mention GeoPandas here but I don't see it being used in the example, is it necessary or is Fiona itself enough? 

Also: we don't document conda in our users installation guide at this moment, so if the necessary dependencies can be installed via pip I'd prefer that.

Reply:
Indeed, if possible pip is preferred. But it used to mix up dependencies between rasterio and fiona/geopandas. Of course, we can simply leave it to pip, and mention users to mind the usage of rasterio in combination with fiona/geopandas. I guess that would indeed be the preferred approach for pydov.

Reply:
Ok, and I need to correct my earlier remark. It's not rasterio that conflicts with fiona/geopandas. It's GDAL's own python bindings. Which was a problem for me I guess, since I was using the OSGEO4W python distribution with the osgeo package.

Reply:
Note that the previous discussion about compatibility of GDAL was about using GDAL's own python bindings and fiona at the same time. In this PR (as far as I can see), you are just using fiona, and not the `gdal` python package. So this is completely fine (and installing fiona both using wheels or using conda is fine). 

The reason that issue came up is because I showed in https://github.com/DOV-Vlaanderen/pydov/issues/132#issuecomment-455212909 a way to get the GML using the `gdal` python bindings.